src/components/templates/AnimateMap/AnimateMap.tsx

```diff
@@ -32,14 +35,20 @@ export const AnimateMap: React.FC<AnimateMapProps> = ({ venue }) => {
 
   useEffect(() => {
     if (!app && dataProvider && containerRef && containerRef.current) {
+      // NOTE: value is either missing or timestamp when user finished their onboarding
+      const firstEntrance = !!user?.onboarded?.perTemplate?.animatemap;
```

Comment:
This is a way to plug in the DB info about user having onboarded the template with the pre-existing animate map game instance expectations of a popup